5.2/5.2.1/debian/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN useradd --system -m -d /plone -U -u 500 plone \
 COPY buildout.cfg /plone/instance/
 
 RUN buildDeps="dpkg-dev gcc libbz2-dev libc6-dev libffi-dev libjpeg62-turbo-dev libopenjp2-7-dev libpcre3-dev libssl-dev libtiff5-dev libxml2-dev libxslt1-dev wget zlib1g-dev" \
- && runDeps="gosu libjpeg62 libopenjp2-7 libtiff5 libxml2 libxslt1.1 lynx netcat poppler-utils rsync wv" \
+ && runDeps="gosu libjpeg62 libopenjp2-7 libtiff5 libxml2 libxslt1.1 lynx make netcat poppler-utils rsync wv" \
```

Comment:
We intentionally removed **gcc/make** from this image in order to make it more lighter and secure. 

99% of the cases you shouldn't need to build libraries from source code within container. For the remaining of 1% we encourage people to create custom Docker images based on this one and do the compilation at the image level.